pipelines/pipeline_katello_nightly.yml

```diff
@@ -11,35 +11,42 @@
   roles:
     - forklift
 
-- hosts: pipeline-katello-nightly-centos7
+- hosts:
+  - pipeline-katello-nightly-centos7
+  - pipeline-proxy-nightly-centos7
   become: yes
   vars:
     katello_repositories_version: nightly
     foreman_repositories_version: nightly
     katello_repositories_use_koji: true
+    foreman_installer_skip_installer: true
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - update_os_packages
+    - foreman_installer
```

Comment:
With master:
```
Wednesday 24 May 2017  16:36:28 +0200 (0:00:00.022)       0:35:33.493 ********* 
=============================================================================== 
foreman_installer : Run installer ------------------------------------- 638.44s
bats : Run bats ------------------------------------------------------- 610.88s
foreman_installer : Install additional packages ----------------------- 267.43s
foreman_installer : Run installer ------------------------------------- 251.64s
foreman_proxy_content : Install Capsule Installer RPM ------------------ 75.53s
foreman_proxy_content : Update packages -------------------------------- 64.10s
foreman_installer : Update packages ------------------------------------ 54.00s
foreman_installer : Install foreman-installer -------------------------- 39.59s
forklift : Bring up boxes ---------------------------------------------- 33.67s
foreman_proxy_content : Generate Certs --------------------------------- 16.01s
forklift : Remove box file --------------------------------------------- 12.42s
foreman_installer : Update packages ------------------------------------- 7.30s
foreman_installer : Install foreman-release-scl ------------------------- 7.17s
epel_repositories : Setup Epel Repository ------------------------------- 4.76s
puppet_repositories : Setup Puppet 4 Repository ------------------------- 4.39s
foreman_installer : Install foreman-release-scl ------------------------- 3.88s
bats : Install git ------------------------------------------------------ 3.74s
foreman_installer : Install foreman-installer --------------------------- 3.06s
epel_repositories : Setup Epel Repository ------------------------------- 2.42s
bats : Run bats --------------------------------------------------------- 2.39s
```

With this branch:
```
Wednesday 24 May 2017  17:11:09 +0200 (0:00:00.031)       0:30:07.720 ********* 
=============================================================================== 
foreman_installer : Run installer ------------------------------------- 635.43s
bats : Run bats ------------------------------------------------------- 370.99s
foreman_installer : Install additional packages ----------------------- 268.77s
foreman_installer : Run installer ------------------------------------- 254.27s
update_os_packages : Update packages ----------------------------------- 71.57s
foreman_installer : Install foreman-installer -------------------------- 48.56s
forklift : Bring up boxes ---------------------------------------------- 43.20s
foreman_proxy_content : Install Capsule Installer RPM ------------------ 34.83s
foreman_proxy_content : Generate Certs --------------------------------- 15.55s
forklift : Remove box file --------------------------------------------- 12.00s
foreman_proxy_content : Update packages --------------------------------- 9.46s
foreman_installer : Install foreman-release-scl ------------------------- 5.06s
puppet_repositories : Setup Puppet 4 Repository ------------------------- 4.57s
epel_repositories : Setup Epel Repository ------------------------------- 2.00s
foreman_installer : Install additional packages ------------------------- 1.89s
foreman_installer : Install foreman-release-scl ------------------------- 1.89s
foreman_installer : Install foreman-installer --------------------------- 1.86s
foreman_installer : Install foreman-release-scl ------------------------- 1.86s
foreman_installer : Install foreman-installer --------------------------- 1.81s
bats : Clone forklift --------------------------------------------------- 1.69s
```

So it shaves off about 5 minutes for me. 2 of those is the ```sleep 120``` in ```fb-install-katello.bats```. I'm guessing that if we could install the additional packages in parallel as well it would shave off some more time.